docs/source/user_guide/installation/index.rst

```diff
@@ -86,7 +98,7 @@ and specific sets of dependencies are listed in the sections below.
 .. _install.plot_dependencies:
 
 Plot dependencies
-^^^^^^^^^^^^^^^^^
```

Comment:
This is needed please put it back

Reply:
will tend to it

Reply:
To clarify, this should be put back for all sections were it changed. Not just this one line

Reply:
Actually they cause inconsistency in header and made the build fail. 
I will look into it .

Reply:
They worked before, is this a linkcheck issue? It is a basic part of restructured text

Reply:
yes, but i resolved your request with my latest commit. sry for the trouble
![image](https://github.com/user-attachments/assets/c78c9777-c93a-419c-bf25-c5bde48f3f75)